src/pages/patientView/clinicalInformation/ClinicalInformationPatientTable.tsx

```diff
@@ -35,15 +35,18 @@ export default class ClinicalInformationPatientTable extends React.Component<
     }): string {
         let ret: string;
 
-        const parsedFloat = parseFloat(data.value);
-        if (parsedFloat != parseInt(data.value) && !Number.isNaN(parsedFloat)) {
-            // the values formatted are not percentages in this case however the function cleanly rounds any float to 1 decimal place
-            ret = formatPercentValue(parsedFloat);
+        if (!isNaN(data.value as any)) {
```

Comment:
@alisman Let me know if this works for you. Doesn't need an outside library and parses before any possible casting.

Reply:
Hi Corey, yes, this is good.    Can you do me one more favor?  Just reverse this if condition, for clarity.  it should be
```
if (isNan(..)) { } else ...
```
We need to run the e2e tests again anyway so might as well improve this.  THANKS!!!!

Reply:
and one more little thing.  far as i can tell, there's no reason to have this local variable ```ret```

where we assign to it, we can just do ```return x```

i see this is legacy btw.

Reply:
@alisman Added updates to adjust for these suggestions. If that's all I think things look good!